core/src/main/java/org/apache/ftpserver/impl/FtpIoSession.java

```diff
@@ -924,7 +924,7 @@ public Certificate[] getClientCertificates() {
      * Update the last-access-time session attribute with the current date
      */
     public void updateLastAccessTime() {
-        setAttribute(ATTRIBUTE_LAST_ACCESS_TIME, new Date());
+        setAttribute(ATTRIBUTE_LAST_ACCESS_TIME, System.currentTimeMillis());
```

Comment:
Given that long is auto-boxed to Long here I don't really see how this would reduce GC churn.